airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/metadata.py

```diff
@@ -92,13 +94,14 @@ def compute_catalog_overrides(merged_df):
     return catalogs
 
 
+# todo change this return value
 def merge_into_metadata_definitions(id_field, connector_type, oss_connector_df, cloud_connector_df) -> pd.Series:
     merged_connectors = pd.merge(
         oss_connector_df, cloud_connector_df, on=id_field, how="outer", suffixes=(OSS_SUFFIX, CLOUD_SUFFIX), indicator=True
     )
     sanitized_connectors = merged_connectors.where(pd.notnull(merged_connectors), None)
```

Comment:
```suggestion
sanitized_connectors = merged_connectors.fillna(None)
```

Reply:
Had to go a slightly different way but thanks for the prompt